qemu/tests/s390x_cpu_model_boot.py

```diff
@@ -0,0 +1,35 @@
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Boot guest with supported cpu-models
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+    boot_cpu_models = params['boot_cpu_models'].split(';')
+    params['start_vm'] = 'yes'
+    vm_name = params['main_vm']
+    vm = env.get_vm(vm_name)
+    args = {'type': 'static', 'model': {'name': 'host'}}
```

Comment:
Could we put it in cfg?

Reply:
cfg:
cpu_model_check = "'query-cpu-model-expansion', {'type': 'static', 'model': {'name': 'host'}}"
py:
host_model = vm.monitor.cmd(cpu_model_check).get('model')

Reply:
solved